### PR TITLE
IF-309: Fix add unit to fields (and IF-340)

### PR DESCRIPTION
**Pull request by jsylupp**

### CODE REVIEW

File: fusionfrontend/src/assets/sass/components/_detailpage.scss [hunk 1]
```diff
@@ -33,13 +36,14 @@
 
   .if-detail-page-div  {
 
-    margin-top: 10px;
+  /*  margin-top: 10px;*/
```
Comment (g1glh0rn) [inline fusionfrontend/src/assets/sass/components/_detailpage.scss:39]: can be removed

**Review by g1glh0rn — CHANGES_REQUESTED** [review]
looks good

**Comment by mattmikulina** [comment]
Task linked: [IF-340 QuantityType "duration" and Unit "s" is automatically assigned to new Metrics/Attributes](https://app.clickup.com/t/2177764/IF-340)

**Comment by mattmikulina** [comment]
Task linked: [IF-309 integrate Unit in Metrics / Attributes](https://app.clickup.com/t/2177764/IF-309)

**Pull request merged**